### PR TITLE
docs(plugins) behavior change in HMAC plugin secret generation

### PR DESCRIPTION
**Pull request by p0pr0ck5**
See kong issue #2143

### CODE REVIEW

File: app/plugins/hmac-authentication.md [hunk 1]
```diff
@@ -78,8 +78,8 @@ $ curl -X POST http://kong:8001/consumers/{consumer}/hmac-auth \
 
 form parameter             | description
 ---                        | ---
-`username`                 | The username to use in the HMAC Signature verification
-`secret`<br>*optional*   | The secret to use in the HMAC Signature verification
+`username`                 | The username to use in the HMAC Signature verification.
+`secret`<br>*optional*   | The secret to use in the HMAC Signature verification. Note that if this parameter isn't provided, Kong will generated a value for you and send it as part of the response body.
```
Comment (theetrain) [inline app/plugins/hmac-authentication.md]: > Kong will `generate` a value...

Comment (p0pr0ck5) [inline app/plugins/hmac-authentication.md]: thanks :)

**Comment by thibaultcha** [comment]
Mashape/kong#2143

**Comment by thibaultcha** [comment]
(plz delete merged branches)

**Pull request merged**